pkg/config/cloudwatch.go

```diff
@@ -7,12 +7,6 @@ import (
 	"github.com/egnyte/ax/pkg/backend/cloudwatch"
 )
 
-func testCloudWatch(em EnvMap) bool {
-	cwClient := cloudwatch.New(em["accesskey"], em["accesssecretkey"], em["region"], "")
-	_, err := cwClient.ListGroups()
-	return err != nil
-}
```

Comment:
@zefhemel You seem to have these for all the backends, but they don't seem to be used. Am I missing something?

Reply:
Right... no, they can be removed.